sonar-csharp-plugin/pom.xml

```diff
@@ -260,7 +260,7 @@
                   </fileset>
                 </copy>
                 <exec executable="dotnet"
-                      dir="${analyzers.directory}/src/SonarAnalyzer.RuleDescriptorGenerator/bin/Release/net6">
+                      dir="${analyzers.directory}/src/SonarAnalyzer.RuleDescriptorGenerator/bin">
```

Comment:
What about calling `dotnet run RuleDescriptorGenerator.csproj --` instead? This builds on demand only and reuses the binaries if there isn't any change AFAIK.

Reply:
I tried, but it has unplesant consequences. It needs to restore and rebuild SonarAnalyzer.Common and SonarAnalyzer.CFG projects as well, because it runs on a separate job from the build :/